_about/notes/0.8.md

```diff
@@ -8,4 +8,35 @@ type: markdown
 ---
 {% include home.html %}
 
-FILL IN BEFORE RELEASE
+In addition to the usual pile of bug fixes and performance improvements, this release includes the new or
+updated features detailed below.
+
+## Networking
```

Comment:
Please add also:  https://github.com/istio/istio/pull/5070

Reply:
we shouldn't highlight the security fixes should we?